### PR TITLE
change pipe default setting

### LINKED ISSUE #311: set default pipe in project settings
Set base R pipe as default in project settings.

### PR DESCRIPTION
**Pull request by chrisorwa**
This PR changes the default pipe to base R.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # serocalculator (development version)
 
 ## New features
+* Change default pipe setting
```
Comment (d-morrison) [inline NEWS.md]: shouldn't this go under the "Internal changes" heading?

Comment (d-morrison) [reply NEWS.md]: and add the PR # in parentheses: `(#312)`

**Review by d-morrison — CHANGES_REQUESTED** [review]
Thanks! Please see comments; minor changes to `NEWS.md` formatting.

File: NEWS.md (hunk 1 above)
Comment (chrisorwa) [reply NEWS.md]: This is well-noted.

File: NEWS.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # serocalculator (development version)
 
 ## New features
+* Change default pipe setting (#312)
```
Comment (d-morrison) [inline NEWS.md]: this change still needs to go under the `## Internal changes` heading (~line 27)

**Review by d-morrison — CHANGES_REQUESTED** [review]
Please see my previous comment about organizing NEWS.md; there are two sections, one for user-facing changes and another for developer-facing changes.

File: NEWS.md (hunk 2 above)
Comment (chrisorwa) [reply NEWS.md]: I have made the change.

**Review by d-morrison — APPROVED** [review]
Looks good, thanks!

**Pull request merged**